tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -101,7 +102,12 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
     #      - task input
     #      - collection configuration
     #      - default value in buckets
-    default_bucket = config.get(CONFIG_COLLECTION_BUCKET_KEY, None)
+    try:
+        default_bucket = config.get(CONFIG_ORCA_DEFAULT_BUCKET_OVERRIDE_KEY)
+    except KeyError:
+        # TODO: Change this to a logging statement
+        print(f"{CONFIG_ORCA_DEFAULT_BUCKET_OVERRIDE_KEY} is not set.")
```

Comment:
Should create a card if we don't have one to do logging in this task instead of print statements.

Maybe state in message using default value? Like `f"{CONFIG_ORCA_DEFAULT_BUCKET_OVERRIDE_KEY} is not set. Using ORCA_DEFAULT_BUCKET value."`

Reply:
That work is in @rizbi-hassan 's PR, so we'll see how the order shakes out. Added extra line.